src/app/course/challenge/course-challenge-snippet/course-challenge-snippet.component.html

```diff
@@ -40,6 +53,19 @@
         </tui-accordion>
     </ng-container>
     <ng-template #skeletonTemplate>
+        <div class="flex justify-end toolBar">
+            <button
+                *ngIf="course?.has_create_event_permission"
+                class="tui-skeleton"
```

Comment:
The skeleton added to Add challenge button is not showing when loading. I'm not sure why.